test_utils/cleanup.go

```diff
@@ -0,0 +1,46 @@
+package testing_init
+
+import (
+	"testing"
+
+	"github.com/port-labs/port-k8s-exporter/pkg/port/blueprint"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/cli"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/page"
+	"github.com/stretchr/testify/assert"
+)
+
+func CheckResourcesExist(portClient *cli.PortClient, t *testing.T, blueprints []string, pages []string) {
+	for _, bp := range blueprints {
+		_, err := blueprint.GetBlueprint(portClient, bp)
+		if err == nil {
+			_ = blueprint.DeleteBlueprint(portClient, bp)
+		}
+		assert.Nil(t, err)
+	}
+
+	for _, p := range pages {
+		_, err := page.GetPage(portClient, p)
+		if err == nil {
+			_ = page.DeletePage(portClient, p)
+		}
+		assert.Nil(t, err)
+	}
+}
+
+func CheckResourcesDoesNotExist(portClient *cli.PortClient, t *testing.T, blueprints []string, pages []string) {
+	for _, bp := range blueprints {
+		_, err := blueprint.GetBlueprint(portClient, bp)
+		if err != nil {
```

Comment:
Why if err != nil (means that the blueprint does not exists), it tries to delete it? Shouldn't be the opposite? The same with pages

Reply:
Yes that's complicated I actually talked it out with @yairsimantov20 to understand, seems like this logic deleted the entities just in case it doesn't gets an error, but fails the test anyway

Reply:
Oh just took another look on this, and yea I get what you mean. Let me fix it :)